packages/core/src/useForm/formContext.ts

```diff
@@ -86,19 +84,23 @@ export function createFormContext<TForm extends FormObject = FormObject, TOutput
   touched,
   snapshots,
 }: FormContextCreateOptions<TForm, TOutput>): BaseFormContext<TForm> {
-  function setFieldValue<TPath extends Path<TForm>>(path: TPath, value: PathValue<TForm, TPath> | undefined) {
+  function setValue<TPath extends Path<TForm>>(path: TPath, value: PathValue<TForm, TPath> | undefined) {
     setInPath(values, path, cloneDeep(value));
   }
 
-  function setFieldTouched<TPath extends Path<TForm>>(path: TPath, value: boolean) {
+  function setTouched<TPath extends Path<TForm>>(path: TPath, value: boolean) {
     setInPath(touched, path, value, true);
   }
```

Comment:
@logaretm how should we handle the case of touching "the whole form".  

we could make `path` optional, but then `value` would need to be optional.

Should we reverse the parameters? (e.g. `setTouched(true, 'myField')` or whole form `setTouched(true)`?

Reply:
We can overload the function:

function setTouched<TPath extends Path<TForm>>(value: boolean)
function setTouched<TPath extends Path<TForm>>(path: string, value: boolean)
function setTouched<TPath extends Path<TForm>>(pathOrValue: TPath, value: boolean) {
  // if first arg is boolean the set whole form, if not then same behavior.
 }

Reply:
I'm embarassed to say I totally forgot you can overload methods in Typescript 😆